recipes/cdutil/build.sh

```diff
@@ -0,0 +1 @@
+python setup.py install
```

Comment:
delete this file in favor of adding `script: python setup.py install` to the `build` section below the `skip` entry.